src/harness/testcases/WINNF_FT_S_PCR_testcase.py

```diff
@@ -342,3 +423,384 @@ def test_WINNF_FT_S_PCR_1(self, config_filename):
+    # SAS does not create a PPA and generates an error.
+    self.assertPpaCreationFailure(ppa_creation_request)
+
+  def generate_PCR_6_default_config(self, filename):
+    """Generate the WinnForum configuration for PCR 6."""
+    # File path where SAS UUT claimed ppa boundary generated in PCR.1 test
+    pcr_1_test_config_file_path = os.path.join('testcases', 'configs',
+                                               'test_WINNF_FT_S_PCR_1',
+                                               'default.config')
+    sas_uut_claimed_ppa_boundary_file_path = getSasUutClaimedPpaBoundaryFilePath(
+        'default.config')
+
+    # Load SAS UUT claimed ppa boundary and check if any error while retrieving
+    # SAS UUT claimed ppa boundary generated in PCR.1 test.
+    try:
+      with open(sas_uut_claimed_ppa_boundary_file_path, 'r') as claimed_ppa_file:
+        user_claimed_ppa_contour = json.load(claimed_ppa_file)
+    except IOError:
+      raise RuntimeError('ConfigError:There is an error in reading path:%s \n\n'
+                         % sas_uut_claimed_ppa_boundary_file_path)
+
+    # Expand the user claimed ppa boundary by 1 kilometer.
+    user_claimed_ppa_contour_shapely = utils.ToShapely(
+      user_claimed_ppa_contour['features'][0]['geometry']).buffer(-1e2)
+    user_claimed_ppa_contour_geometry = json.loads(utils.ToGeoJson(
+      user_claimed_ppa_contour_shapely))
+
+    # Create the actual config.
+    config = {
+        'configPCR_1': pcr_1_test_config_file_path,
+        'userClaimedPpaContour': user_claimed_ppa_contour_geometry
+    }
+    writeConfig(filename, config)
+
+  @configurable_testcase(generate_PCR_6_default_config)
+  def test_WINNF_FT_S_PCR_6(self, config_filename):
+    """Unsuccessful PPA boundary Claimed by PAL Holder Not contained within Maximum PPA Boundary.
+
+    SAS UUT shall reject a PPA boundary claimed by the PAL holder,
+    that is not fully contained within the maximum PPA boundary created by SAS UUT.
+    """
+    # Load the Config file.
+    config = loadConfig(config_filename)
+
+    # Load the test_WINNF_FT_S_PCR_1 config. All other inputs must be identical
+    # to those used in the corresponding configuration of PCR.1.
+    pcr_1_test_config = loadConfig(config['configPCR_1'])
+
+    # Inject the PAL records.
+    for pal_record in pcr_1_test_config['palRecords']:
+      self._sas_admin.InjectPalDatabaseRecord(pal_record)
+
+    # Register devices and  check response.
+    cbsd_ids = self.assertRegistered(pcr_1_test_config['registrationRequests'],
+                                     pcr_1_test_config['conditionalRegistrationData'])
+
+    # Trigger SAS UUT to create a PPA boundary.
+    pal_ids = [record['palId'] for record in pcr_1_test_config['palRecords']]
+
+    # Check if the PPA generated by the SAS UUT is fully contained within
+    # the service area.
+    self.assertTrue(isPpaWithinServiceArea(pcr_1_test_config['palRecords'],
```

Comment:
I guess this is to catch config errors? If yes, please make that clear in the error msg. If not, please explain :)